test/__tests__/data/ga4.json

```diff
@@ -10140,635 +10140,6 @@
       "files": {}
     }
   },
```

Comment:
We have removed the identify test cases only. Let's add some test cases with new track events.

Reply:
added test cases for login, sign_up and generate_lead event